client/app/visualizations/funnel/Renderer/index.jsx

```diff
@@ -16,7 +16,7 @@ function generateRowKeyPrefix() {
 
 export default function Renderer({ data, options }) {
   const funnelData = useMemo(() => prepareData(data.rows, options), [data, options]);
-  const rowKeyPrefix = useMemo(() => generateRowKeyPrefix(), [funnelData]);
+  const rowKeyPrefix = useMemo(() => generateRowKeyPrefix(), []);
```

Comment:
`rowKeyPrefix` should be generated every time `funnelData` changes to ensure that Ant Table will properly update rows

Reply:
I was checking, `useEffect` doesn't require dependencies to be used (`useMemo` does), so the options: 1 - Move it to `useEffect` and `useState` for the rowKeyPrefix; 2 - eslint-disable;

I think 2 is the quick one ^^

Reply:
1 is more confusing (IMO) - this use case is perfect for `useMemo`, but not as obvious with `useState` + `useEffect`. So I'm for 2, or I can try to find another (?) solution.